.env.test

```diff
@@ -1,18 +1,7 @@
 FLASK_APP=todo_app/app
 FLASK_ENV=development
 FLASK_SKIP_DOTENV=True
-
-SECRET_KEY = "491a167e67abe9aa3d40e1e4"
-
-token = "4c82ed8e16b482daf8c417cc0f1187bf943dc7d3dcbd212d08b39b51a5c58847"
-api_key = "38562d78c91237d032fc48f4301cab52"
-TRELLO_BOARD_ID = "34"
-FLASK_APP=todo_app/app
-FLASK_ENV=development
-FLASK_SKIP_DOTENV=True
-
-SECRET_KEY=491a167e67abe9aa3d40e1e4
-
-token=4c82ed8e16b482daf8c417cc0f1187bf943dc7d3dcbd212d08b39b51a5c58847
-api_key=38562d78c91237d032fc48f4301cab52
-TRELLO_BOARD_ID=34
\ No newline at end of file
+SECRET_KEY="491a167e67abe9aa3d40e1e4"
+token="4c82ed8e16b482daf8c417cc0f1187bf943dc7d3dcbd212d08b39b51a5c58847"
+api_key="38562d78c91237d032fc48f4301cab52"
```

Comment:
Btw, this is not a python file, so you don't need quotes around the values. In fact, when passing it to Docker with `--env-file`, you need to NOT have the quotes or else it includes them in the actual value.

So check that your .env file doesn't use quotes.